### PR TITLE
Update config.md

### PR DESCRIPTION
**Pull request by AugustoAmaral**
Just a minor change that can be usefull

### CODE REVIEW

File: _includes/rest/config.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The response body is a JSON object containing all the configuration parameters i
 }
 ```
 
-You can also update the config by sending a `PUT` request to config URL. Here is a simple example that will update the `Parse.Config`:
+You can also update the config by sending a `PUT` request to config URL. Here is a simple example that will update the `Parse.Config` (requires masterKey) :
```
Comment (TomWFox) [inline _includes/rest/config.md]:
```suggestion
You can also update the config by sending a `PUT` request to config URL. Here is a simple example that will update the `Parse.Config` (requires `masterKey`):
```

File: _includes/rest/config.md [hunk 2]
```diff
@@ -70,3 +70,5 @@ The response body is a JSON object containing a simple boolean value in the `res
   "result": true
 }
 ```
+
+If you want to make any changes on configs without sending the masterkey,  you will need to create a cloud function that makes those changes.
```
Comment (TomWFox) [inline _includes/rest/config.md]:
```suggestion
If you want to make any changes to configs without sending the `masterkey`, you will need to create a Cloud Function that makes those changes.
```

**Review by TomWFox — CHANGES_REQUESTED** [review]
Thanks for making a PR!

Seems like a useful note to add, just a couple minor changes needed.

**Comment by AugustoAmaral** [comment]
Done

**Review by TomWFox — APPROVED** [review]
Looks good, thanks for contributing!

**Pull request merged**